<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>StuWindow</class>
 <widget class="QMainWindow" name="StuWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>752</width>
    <height>442</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>学生用户界面</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>查看已选课程信息</string>
       </attribute>
       <widget class="QTableView" name="tableView_g_2">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>50</y>
          <width>651</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>30</x>
          <y>10</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>已选课程</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>选退课</string>
       </attribute>
       <widget class="QTableView" name="tableView_c">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>40</y>
          <width>631</width>
          <height>261</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label">
        <property name="geometry">
         <rect>
          <x>40</x>
          <y>10</y>
          <width>54</width>
          <height>16</height>
         </rect>
        </property>
        <property name="text">
         <string>可选课程</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_add">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>320</y>
          <width>80</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>加入选中课程</string>
        </property>
       </widget>
       <widget class="QPushButton" name="pushButton_exit">
        <property name="geometry">
         <rect>
          <x>400</x>
          <y>320</y>
          <width>81</width>
          <height>24</height>
         </rect>
        </property>
        <property name="text">
         <string>退出选中课程</string>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_8">
       <attribute name="title">
        <string>成绩显示</string>
       </attribute>
       <widget class="QTableView" name="tableView_g">
        <property name="geometry">
         <rect>
          <x>80</x>
          <y>20</y>
          <width>561</width>
          <height>281</height>
         </rect>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="tab_3">
       <attribute name="title">
        <string>通知查看</string>
       </attribute>
       <widget class="QTableView" name="tableView_i">
        <property name="geometry">
         <rect>
          <x>9</x>
          <y>9</y>
          <width>311</width>
          <height>341</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>9</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QTextEdit" name="textEdit_i">
        <property name="geometry">
         <rect>
          <x>333</x>
          <y>9</y>
          <width>391</width>
          <height>341</height>
         </rect>
        </property>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>752</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
